README.md

```diff
@@ -89,32 +106,30 @@ apply the snapshot to get the latest state of the chain. The downside here
 is the fact that your node won't have the chain history prior to the snapshot.
 Moreover, you need to trust the source of the snapshot.
 
-Mezo team provides snapshots for Mezo Matsnet testnet. Please refer to
-[this runbook](./manual/README.md#State-sync-from-snapshot)
-for details. Alternatively, you can ask trusted community members for a snapshot.
+Mezo team provides snapshots only for Mezo Matsnet testnet. Mezo team **DOES NOT** 
+provide snapshots for Mezo Mainnet. In any case, you can ask trusted community members
```

Comment:
Once we chose 3 validators to become RPC and seed notes, it would be good to recommend them here. I'm pretty sure new validators will ask a question on who can provide a snapshot and having this info in the docs will reduce support time.

Reply:
Created an issue https://github.com/mezo-org/validator-kit/issues/61